config_loader/metadata.rb

```diff
@@ -0,0 +1,23 @@
+name 'config_loader'
+maintainer 'The Authors'
+maintainer_email 'you@example.com'
+license 'All Rights Reserved'
+description 'Installs/Configures config_loader'
+long_description 'Installs/Configures config_loader'
+version '0.2.2'
+chef_version '>= 12.15.19' if respond_to?(:chef_version)
+
+depends 'citadel_fork'
+depends 'aws_metadata'
```

Comment:
Just a note to ourselves: the caller of this cookbook will have to ensure that these are both in the toplevel Berksfile.